### ISSUE TITLE
No Speech Feedback Changing Durations in Normal Mode

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#30482]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

In 4.6 there is speech feedback when you select a new duration option in Note Input mode, but no feedback in Normal mode when the note is actually changed.

1.  Press N to make sure you are in Normal mode.
2. With focus on any note or rest press a number key to change that note value.  Notice the note value changes but there is no speech feedback.  The user has to arrow on and off the note to see that it has in fact changed.
3. Press W or Q to half/double the note duration.  Again notice that the note changes but there is no speech feedback from the screen reader to announce the change.
4. Press N to go to Note Input by Letter mode and repeat the above steps.  Notice that this time as each input type is changed the screen reader announces the duration.

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.6.1

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
